packages/api/src/controllers/sensorController.js

```diff
@@ -231,6 +231,34 @@ const sensorController = {
     }
   },
 
+  async getReadingsByFarmId(req, res) {
+    try {
+      const { farm_id, days } = req.params;
+      if (!farm_id) {
+        res.status(400).send('Invalid farm id');
+      }
+      let result;
+      if (!days) {
+        result = await SensorReadingModel.query()
+          .joinRaw('JOIN sensor ON sensor_reading.sensor_id::uuid = sensor.sensor_id')
+          .where('farm_id', farm_id)
+          .debug();
```

Comment:
Is this required? cannot find it anywhere else in the project

Reply:
sensor_id is uuid in sensor_reading table but it's string in sensor table so we need to type cast